snuba/query/snql/__init__.py

```diff
@@ -46,8 +46,10 @@
     having_clause         = space* "HAVING" or_expression space*
     order_by_clause       = space* "ORDER BY" order_list space*
 
-    condition             = low_pri_arithmetic condition_op (function_call / column_name / numeric_literal / string_literal) space*
+    main_condition        = low_pri_arithmetic condition_op (function_call / column_name / numeric_literal) space*
```

Comment:
Why did you remove the string_literal ?

(Which by the way should be a quoted_literal)

Reply:
Added quoted_literal.